.github/workflows/build_static.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: x86_64-unknown-linux-musl-services
-          path: target/x86_64-unknown-linux-musl/release
+          path: target/release
```

Comment:
To make the path in the Dockerfile independent of the architecture.

Reply:
Sounds good but I don't see the corresponding change that changes the output path for Cargo. E.g. `crossShell` and `cross-shell.nix` are still the same

Reply:
I think the way github zips up files is that it finds the lowest common parent directory and strips the other directories. So changing the extraction directory here works for the CI. It currently wouldn't work for `scripts/build-docker-images`, or it would now package normal binaries not the statically linked binaries. I also still have to test if the containers built like this actually work.

Reply:
I see. That is very opaque on GitHub's part lol. But this is fine as long as the images work